lib/bugsnag/integrations/railtie.rb

```diff
@@ -69,6 +69,7 @@ class Railtie < Rails::Railtie
           end
         end
       end
+      Bugsnag.configure(true) # Validate API key is set
```

Comment:
You dont even need this because the previous configure will also validate?

Reply:
There isn't another configure call if the generator was never run.

Reply:
The one above I mean?

Reply:
Oh you mean the immediately preceding line? Yes, that would be more logical.